FILES.deploy-tools/root/inventory/minimal-for-testing

```diff
@@ -6,7 +6,25 @@ logical_cluster_name=example
 flavor=m1.medium
 
 #
+<<<<<<< a5dc9d8e2c46692d4269eae32a5f7e8127571a63
 # inventory customizations
+=======
+apiserver_image_name=ndslabs/apiserver:latest
+webui_image_name=ndslabs/angular-ui:latest
+loadbal_image_name=gcr.io/google_containers/nginx-ingress-controller:0.8.2
+gluster_image_name=ndslabs/gluster:latest
+nrpe_image_name=ndslabs/nagios-nrpe:latest
+backup_image_name=davidraila/cluster-backup:NDS-348
+
+#
+# backup vars
+#
+backup_user=centos
+backup_host=132.249.238.215
+backup_path=/ndsbackup
+backup_key=backup.pem
+
+>>>>>>> add cluster backup
```

Comment:
This appears to be a merge conflict... Either fix the conflict or remove the file altogether, since it is no longer used